contracts/TradeLedger.sol

```diff
@@ -397,7 +397,7 @@ contract TradeLedger is Owned {
   ) 
     constant
     accountPresent(accountId) // Only valid accounts can be provided
-    returns (string, int256, int256, int256, int256)
+    returns (uint256, string, int256, int256, int256, int256, string)
```

Comment:
Is this tested? Seemed to get some weird overflow error when adding too many return values here